patches/minecraft/net/minecraft/world/inventory/BrewingStandMenu.java.patch

```diff
@@ -1,5 +1,14 @@
 --- a/net/minecraft/world/inventory/BrewingStandMenu.java
 +++ b/net/minecraft/world/inventory/BrewingStandMenu.java
+@@ -75,7 +_,7 @@
+                if (!this.m_38903_(itemstack1, 3, 4, false)) {
+                   return ItemStack.f_41583_;
+                }
+-            } else if (BrewingStandMenu.PotionSlot.m_39133_(itemstack) && itemstack.m_41613_() == 1) {
```

Comment:
Does this have other concequences?

Reply:
As far as I can tell, no. The `m_38903_` calls a safe insert method which handles proper splitting based on the slot's maximum size. So, there are no standing consequences in relation to the single item slot output. The only major consequence would be if someone tried to modify the brewing stand to take in more than one input; however, that would be on the responsibility of the modder to fix that issue as they tried to coremod that existence.